linkerd/app/inbound/src/endpoint.rs

```diff
@@ -78,20 +116,23 @@ impl tls::HasPeerIdentity for HttpEndpoint {
     }
 }
 
-// === TcpEndpoint ===
-
-impl From<listen::Addrs> for TcpEndpoint {
-    fn from(addrs: listen::Addrs) -> Self {
-        Self {
-            port: addrs.target_addr().port(),
-        }
+impl Into<transport::labels::Key> for &'_ HttpEndpoint {
+    fn into(self) -> transport::labels::Key {
+        transport::labels::Key::Connect(transport::labels::EndpointLabels {
+            direction: transport::labels::Direction::In,
+            authority: None,
+            labels: None,
+            tls_id: tls::Conditional::None(tls::ReasonForNoPeerName::Loopback.into()).into(),
+        })
```

Comment:
~~nit/tioli: maybe there should be a `Default` impl for `EndpointLabels` so we don't have to write out the `None`s?~~

actually, hmm, that might be a bit weird since it would mean picking a `Direction` to default to, and then always clobber it...never mind, maybe this isn't great?